idd/Energy+.idd.in

```diff
@@ -19458,6 +19458,7 @@ RoomAir:Node:AirflowNetwork:InternalGains,
       \key ElectricLoadCenter:Inverter:LookUpTable
       \key ElectricLoadCenter:Storage:Battery
       \key ElectricLoadCenter:Storage:Simple
+      \key ElectricLoadCenter:Storage:Converter
```

Comment:
OK, I'll push a commit to this branch without those idd changes and push them up in a separate branch immediately after.  This will slide right in.

@kbenne @macumber I'm still curious about whether there is a list of acceptable changes/unacceptable changes to the IDD after the IO freeze.  If the freeze is 100% frozen except SHA, then so be it, just curious if there is a clean list.